components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -386,13 +386,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
   }
 
   onFiltersClear(_event) {
-    const queryParams = {...this.route.snapshot.queryParams};
-
-    const filteredParams = pickBy((_value, key) => {
-        return this.allowedURLFilterTypes.indexOf(key) < 0;
-      }, queryParams);
-
-    this.router.navigate([], {queryParams: filteredParams});
```

Comment:
If feels like maybe I'm missing something here - or maybe this served a dual purpose in the past?

Since we are clearing ALL the applied filters, it makes sense to me that if we send through an empty object, we'll clear all the applied filters, instead of looping through the filters one at a time to parse them into an object and then remove them.

Reply:
@msorens @Shadae - would love your thoughts on this when you have a chance

Reply:
is it maybe that we were clearing all filters _except_ end time/start time?  i noticed those values aren't in the allowedURLFilterTypes function..and those are "default" filters

Reply:
@vjeffrey ahhh Yes! thats it, thank you!  It doesn't add a date filter to the URL if not explicitly selecting a date other than today